modules/nw-cluster-mtu-change.adoc

```diff
@@ -0,0 +1,309 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-network-mtu.adoc
+
+[id="nw-cluster-mtu-change_{context}"]
+= Changing the cluster MTU
+
+As a cluster administrator, you can change the maximum transmission unit (MTU) for your cluster. The migration is disruptive and nodes in your cluster might be temporarily unavailable as the MTU update rolls out.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+* You identified the target MTU for your cluster. The correct MTU varies depending on the cluster network provider that your cluster uses:
+** *OVN-Kubernetes*: The cluster MTU must be set to `100` less than the lowest MTU value in your cluster.
+** *OpenShift SDN*: The cluster MTU must be set to `50` less than the lowest MTU value in your cluster.
+
+.Procedure
+
+To increase or decrease the MTU for the cluster network complete the following procedure.
+
+. To obtain the current MTU for the cluster network, enter the following command:
++
+[source,terminal]
+----
+$ oc describe network.config cluster
+----
+
+. Get the NetworkManager connection name:
+
+.. To find the primary network interface, enter the following command:
++
+[source,terminal]
+----
+$ oc debug node/<node_name> -- chroot /host ip route list match 0.0.0.0/0 | awk '{print $5 }'
+----
++
+--
+where:
+
+`<node_name>`:: Specifies the name of a node in your cluster.
+--
+
+.. To find the connection that NetworkManager created for the interface name returned from the previous command, enter the following command:
++
+[source,terminal]
+----
+$ oc debug node/<node_name> -- chroot /host nmcli c | grep $(ip route list match 0.0.0.0/0 | awk '{print $5 }')
+----
++
+.Example output
+[source,text]
+----
+Wired connection 1  46da4a6a-xxxx-xxxx-xxxx-ac0ca900f213  ethernet  ens3
```

Comment:
This is not used later in the doc for anything?

Reply:
@jcaamano, this is the NetworkManager profile name that's used later for a user that wants to use this to set the MTU. But maybe it makes sense to do this elsewhere.

Reply:
But I don't see how this is used later in the document?

Reply:
Looks like it doesn't work, at least locally. I lack the skill to determine how to fix it.

```
oc debug node/crc-8rwmc-master-0 -- chroot /host nmcli c | grep $(ip route list match 0.0.0.0/0 | awk '{ print $5 }')
Starting pod/crc-8rwmc-master-0-debug ...
To use host binaries, run `chroot /host`

Removing debug pod ...
```

```
oc debug node/crc-8rwmc-master-0 -- chroot /host ip route list match 0.0.0.0/0 | awk '{ print $5 }'
Starting pod/crc-8rwmc-master-0-debug ...
To use host binaries, run `chroot /host`
enp1s0

Removing debug pod ...
```

```
oc debug node/crc-8rwmc-master-0 -- chroot /host nmcli c | grep $(echo 'enp1s0')
Starting pod/crc-8rwmc-master-0-debug ...
To use host binaries, run `chroot /host`
Wired connection 1  3e96a02b-88e0-3707-ad5d-61db28678130  ethernet  enp1s0

Removing debug pod ...
```

But somehow this is possible.

Reply:
I am confused about where that command comes from, but yeah, that won't work. Divide it in two:

```
dev=$(oc debug node/crc-8rwmc-master-0 -- ip route show default | awk '{ print $5; exit }')
profile=$(oc debug node/crc-8rwmc-master-0 -- chroot /host nmcli -g DEVICE,CON-UUID d | grep $dev | cut -d: -f2)
```

Reply:
@jcaamano, I did something similar and moved the step; It only seems applicable for the MachineConfig approach.